pkgs/development/python-modules/multivolumefile/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  setuptools,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "multivolumefile";
+  version = "0.2.3";
+
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-oGSNCq+8luWRmNXBfprK1+tTGr6lEDXQjOgGDcrXCdY=";
+  };
+
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
+
+  pythonImportsCheck = [ "multivolumefile" ];
+
+  doCheck = true;
```

Comment:
No nativeCheckInputs?

Reply:
Getting this when trying to run native checks:

```
error: some outputs of '/nix/store/dbybz5crs22r5k71kwyi8ccn75mkmf33-python3.12-multivolumefile-0.2.3.drv' are not valid, so checking is not possible
```

Reply:
User error. Fixed.